examples/generalized-compute/extract-dicom-tags/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# !! EDIT THIS: Set base Docker image to be used.
```

Comment:
This folder is missing a README.md file to explain what this example is, how to use it, etc.